app/controllers/items_controller.rb

```diff
@@ -204,14 +204,14 @@ def embargo_update
   end
 
   def datastream_update
-    ds = @object.datastreams[params[:dsid]]
-    # check that the content is valid xml
     begin
-      content = Nokogiri::XML(params[:content]){ |config| config.strict }
+      # check that the content is well-formed xml
+      Nokogiri::XML(params[:content]) { |config| config.strict }
     rescue
```

Comment:
Is it possible to rescue the specific error here? Is it always going to be `Nokogiri::XML::SyntaxError`?